pkg/services/object/internal/client/client.go

```diff
@@ -316,6 +317,10 @@ func (x PayloadRangeRes) PayloadRange() []byte {
 	return x.data
 }
 
+// maxInitialBufferSize is the maximum initial buffer size for PayloadRange result.
+// On one side we don't want to reallocate a lot, on the other --
```

Comment:
what does `--` mean?

Reply:
It is a dash.

Reply:
Forgot another line, fixed the description.